Bindings/Java/Matlab/Hopper_Device/BuildHopper.m

```diff
@@ -24,26 +24,29 @@
 % Build a model of a one-leg hopper, with one muscle.
 
 p = inputParser();
-defaultMuscleModel = 'Thelen2003';
+defaultMuscleModel = 'Millard2012Equilibrium';
 defaultMaxIsometricForce = 4000.0;
 defaultMillardTendonParams = [0.049 28.1 0.67 0.5 0.25];
-defaultActivation = [0.0 1.99 2.0 3.89 3.9 4.0;
+defaultExcitation = [0.0 1.99 2.0 3.89 3.9 4.0;
                      0.3 0.3  1.0 1.0  0.1 0.1];
-defaultPrintModel = false; 
+defaultPrintModel = false;
+defaultAdditionalMass = 0;
```

Comment:
I'm going to create a function called InteractiveHopperSettings, that, given a name for a device, muscle, or anything other built-in settings we want, can be called to set those values. I would leave it inside RunInteractiveHopperSolution, except that it would need to be called by InteractiveHopper to update the values shown in the GUI. If we want to cut down the number of files and come up with a better solution, we could figure out a way to keep it in RunInteractiveHopperSolution. It could be nice to keep the settings in a separate function, rather than having to go to the bottom of RunInteractiveHopperSolution.